tests/__init__.py

```
@@ -172,7 +222,13 @@ def loadDocstringsFromModule(module):
 		raise ImportError("[CWE-440] doctest Failed to import.") from _cause
 	finder = doctest.DocTestFinder(verbose=True, recurse=True, exclude_empty=True)
 	doc_suite = unittest.TestSuite()
-	doc_suite.addTests(doctest.DocTestSuite(module=module, test_finder=finder))
+	try:
+		doc_suite.addTests(doctest.DocTestSuite(module=module, test_finder=finder))
+	except ValueError as e:
+		# ValueError is raised when no tests are found
+		print(f"No doctests found in {module.__name__}: {e}")
+	except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)